style.css

```diff
@@ -0,0 +1,26 @@
+#imge {
+  text-align: center;
+}
+#title {
+  text-align: center;
+}
+p {
+  text-align: center;
+}
+h2 {
+  text-align: center;
+}
```

Comment:
- [ ] Kindly note to always include an explanation, a "why", for each requested change. For example:

This is a well-explained request:
- Following the DRY rule (Don't repeat yourself), it would be better to apply the style to all selectors by separating them with commas.  You can read more about this rule on this [link](https://en.wikipedia.org/wiki/Don't_repeat_yourself).
This request lacks a "why":
- I think it would be better to apply the style to all selectors by separating them with comas,